update.sh

```diff
@@ -145,5 +145,17 @@ for version in "${versions[@]}"; do
 			-e "s!%%INSTALL_PYTHON%%!$installPython!g" \
 			Dockerfile-management.template \
 			> "$version/$variant/management/Dockerfile"
+
+		# rabbitmq_prometheus is only included in 3.8+
+		# same with "/etc/rabbitmq/conf.d" support (https://github.com/rabbitmq/rabbitmq-server/pull/2258, https://github.com/rabbitmq/rabbitmq-server/pull/2277)
+		if [ "$rcVersion" = '3.7' ]; then
```

Comment:
Now that https://github.com/docker-library/rabbitmq/pull/431 is merged, if we revert all these changes to `update.sh`, we can rebase on top of the current master, merge & ship it.

Reply:
Nice catch, updated! :+1: